### PR TITLE
Fix BotStudio sliders on mac OS

### PR DESCRIPTION
**Pull request by omichel**
Fixed the ability to inverse the condition (">" or "<") with the sliders of the distance sensors for the e-puck robot in the BotStudio window on mac OS.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:eyes:

File: projects/robots/gctronic/e-puck/plugins/robot_windows/botstudio/e-puck/EPuckSlider.cpp [hunk 1]
```diff
@@ -186,6 +192,24 @@ EPuckSlider::~EPuckSlider() {
...
+void EPuckSlider::sliderMoved(int value) {
+  mWasDragged = true;
+  mPreviousValue = value;
+}
+
+void EPuckSlider::sliderReleased() {
+  if (mType == RevertibleSlider && !mWasDragged) {
+    const int v = mSlider->value();
+    if (v == mPreviousValue)
+      setInverted(!isInverted());
```
Comment (stefaniapedrazzi) [inline projects/robots/gctronic/e-puck/plugins/robot_windows/botstudio/e-puck/EPuckSlider.cpp:208]: Would not be better to also call `update()` here?

Comment (omichel) [reply projects/robots/gctronic/e-puck/plugins/robot_windows/botstudio/e-puck/EPuckSlider.cpp:208]: I just added it, but it doesn't change anything to the behavior...

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you

**Pull request merged**